foundation/foundation-mda/src/test/java/com/boozallen/aiops/mda/metamodel/element/AbstractModelInstanceSteps.java

```diff
@@ -284,17 +284,28 @@ protected void createProject(String projectName, String moduleName, String url)
         writePom(model, projectDir);
     }
 
-    protected void createPipeline(String name, String typeName, String implName) throws IOException {
-        createPipeline(name, typeName, implName, p -> {});
+    protected void createAndSavePipeline(String name, String typeName, String implName) throws IOException {
+        createAndSavePipeline(name, typeName, implName, p -> {});
     }
 
-    protected void createPipeline(String name, String typeName, String implName, Consumer<PipelineElement> customization) throws IOException {
+    protected void createAndSavePipeline(String name, String typeName, String implName, Consumer<PipelineElement> customization) throws IOException {
         this.pipeline = TestMetamodelUtil.createPipelineWithType(name, BOOZ_ALLEN_PACKAGE, typeName, implName);
         customization.accept(pipeline);
         savePipelineToFile(pipeline);
         pipelines.put(name, pipeline);
```

Comment:
Q: Could this method call the `createPipeline` method below instead of duplicating the code:
```
this.pipeline = TestMetamodelUtil.createPipelineWithType(name, BOOZ_ALLEN_PACKAGE, typeName, implName);
pipelines.put(name, pipeline);
```

Reply:
Actually, I've found it difficult to access the pipeline object outside of this context.

Reply:
Refactored.